internal/storage/coin_storage.go

```diff
@@ -245,6 +250,15 @@ func (c *CoinStorage) AddingBlock(
 ) (CommitWorker, error) {
 	for _, txn := range block.Transactions {
 		for _, operation := range txn.Operations {
+			success, err := c.asserter.OperationSuccessful(operation)
+			if err != nil {
+				return nil, fmt.Errorf("%w: unable to parse operation success", err)
```

Comment:
how about
```suggestion
				return nil, fmt.Errorf("%w: failed to parse operation", err)
```

Reply:
I'd actually like to keep this as is. Operation parsing occurs before it reaches this loop and in this case we are very specifically parsing `*types.OperationSuccess`.

Reply:
ah `operation success` means `operation with success status`. Maybe want to more explicit since it's not super clear when I read it first time